scripts/docs.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+set -u
+
+if [ -z `which jazzy` ]; then
+    echo "Installing jazzy…"
+    gem install jazzy
+    if [ -z `which jazzy` ]; then
+        echo "Unable to install jazzy."
+        exit 1
+    fi
+fi
+
+for module in MapboxNavigation MapboxCoreNavigation
+do
+    jazzy \
+        --config jazzy.yml \
+        --module "$module" \
+        --module-version 0.4.0 \
+        --root-url 'https://www.mapbox.com/navigation-sdk/ios/0.4.0/' \
```

Comment:
I'm not sure I understand why it would drop the `/0.4.0/` even though it's specified in explicitly in `--root-url`. Can you elaborate @1ec5?